tor/core/inbox.py

```diff
@@ -22,12 +22,15 @@
 
 
 def forward_to_slack(item, config):
-    send_to_slack(
-        f'Unhandled inbox reply by *{item.author}* -- *{item.subject}*: '
-        f'{item.body}', config)
+    username = item.author.name
 
+    send_to_slack(
+        f'Unhandled message by '
+        f'<https://reddit.com/user/{username}|u/{username}> -- '
```

Comment:
Why do you have the `{username}|u/{username}`? If it's for visual reasons, can you move the greater than sign to be after the first `{username}`?

Reply:
https://github.com/GrafeasGroup/tor/blob/master/tor/core/user_interaction.py#L53 This is how slack formats links. The first part is the actual link, the second part is how you want it to appear in slack. It's valid.

Reply:
Figured